lexer/lexer.go

```diff
@@ -90,6 +90,18 @@ func (l *Lexer) NextToken() token.Token {
 		l.skipToNextLine()
 		l.skipWhitespace()
 	}
+	
+	// Check for multi-line comment.
+	// Starts with /* and ends with */.
+	// /* and */ must be the only characters on their line
+	for (l.ch == '/' && l.peekChar() == '*') {
+		for (l.ch !=  '*' && l.peekChar() != '/'){
+			l.skipToNextLine()
+			l.skipWhitespace()
+		}
+		l.skipToNextLine()
+		l.skipWhitespace()
+	}
 
```

Comment:
Please add new tests for multiline comments in both `lexer/lexer_test.go` and `emitter/emitter_test.go`.  For the emitter tests, you can simply sprinkle in some multiline comments to existing tests and ensure they still pass.  You can run all the tests with:
```
// From the root directory
go test ./...
```

Reply:
I added some emitter tests aswell as a few lexer ones. I'm not sure how to test if an error triggers if a comment isn't closed on both though.